go/cmd/dolt/commands/schcmds/tags.go

```diff
@@ -87,24 +88,63 @@ func (cmd TagsCmd) Exec(ctx context.Context, commandStr string, args []string, d
 			return 0
 		}
 	}
-	for i := 0; i < len(tables); i++ {
-		tables[i] = fmt.Sprintf("'%s'", tables[i])
+
+	var headerSchema = sql.Schema{
+		{Name: "table", Type: sql.Text, Default: nil},
+		{Name: "column", Type: sql.Text, Default: nil},
+		{Name: "tag", Type: sql.Uint64, Default: nil},
+	}
+
+	rows := make([]sql.Row, 0)
+
+	for _, tableName := range tables {
+		table, foundTableKey, ok, err := root.GetTableInsensitive(ctx, tableName)
+
+		// Return an error if table is not found
+		if !ok {
+			return commands.HandleVErrAndExitCode(errhand.BuildDError("Can't find table %s.", tableName).AddCause(err).Build(), usage)
+		}
+
+		if err != nil {
+			return commands.HandleVErrAndExitCode(errhand.BuildDError("Could not load table %s.", tableName).AddCause(err).Build(), usage)
+		}
+
+		sch, err := table.GetSchema(ctx)
+
+		if err != nil {
+			return commands.HandleVErrAndExitCode(errhand.BuildDError("Could not load %s schema.", tableName).AddCause(err).Build(), usage)
+		}
+
+		_ = sch.GetAllCols().Iter(func(tag uint64, col schema.Column) (stop bool, err error) {
+			rows = append(rows, sql.NewRow(foundTableKey, col.Name, tag))
+
+			return false, err
+		})
+
+	}
+
+	outputFmt, verr := commands.GetResultFormat("tabular")
```

Comment:
No need to call GetResultFormat here, just assign the var with the tabular const value directly